src/quo2/components/switchers/group_messaging_card/component_spec.cljs

```diff
@@ -0,0 +1,116 @@
+(ns quo2.components.switchers.group-messaging-card.component-spec
+  (:require [test-helpers.component :as h]
+            [quo2.components.switchers.group-messaging-card.view :as group-messaging-card]
+            [quo2.components.switchers.utils :as utils]))
+
+(defonce photos-list
+  [{:source (js/require "../resources/images/mock2/photo1.png")}
+   {:source (js/require "../resources/images/mock2/photo2.png")}
+   {:source (js/require "../resources/images/mock2/photo3.png")}
+   {:source (js/require "../resources/images/mock2/photo1.png")}
+   {:source (js/require "../resources/images/mock2/photo2.png")}
+   {:source (js/require "../resources/images/mock2/photo3.png")}])
+(defonce sticker {:source (js/require "../resources/images/mock2/sticker.png")})
```

Comment:
Nit: `defonce` is more of a REPL thing to avoid side-effects when evaluating namespaces or the individual def* expressions. In these component spec namespaces on the other hand, they only run successfully via the slow compilation to files, which means, changing the file always cause a complete recompilation and the previous state is lost anyway.

Reply:
So is it still better to use just `def`?

Reply:
I would say so @codemaster115. Reserve `defonce` for those cases where reloading in a REPL session can cause unwanted side-effects, otherwise `def` is enough.

Reply:
Got it. Thanks, @ilmotta